scripts/build/ci-deploy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM circleci/golang:1.11
+FROM circleci/golang:1.13
```

Comment:
Due to changes in this file you'll need to update/increment  version and push to docker hub, see .https://github.com/grafana/grafana/blob/master/scripts/build/ci-deploy/build-deploy.sh When that's done you want to update all references to grafana/grafana-ci-deploy in the repository.